Azure/cloud-init.txt

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description: This script will install all dependencies
+# required by clear containers and metrics/tests.
+#
+# OS: ubuntu 16.04
+
+# Add repository key
+wget -nv https://download.opensuse.org/repositories/home:clearcontainers:clear-containers-3/xUbuntu_16.04/Release.key -O Release.key
+apt-key add - < Release.key
+apt-get update
+
+# Add repositories and install runtime
+echo "Installing Clear Containers"
+sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_16.04/ /' > /etc/apt/sources.list.d/cc-runtime.list"
+apt-get update
+apt-get install -y cc-runtime
+
+
+# Install docker
+echo "Installing docker"
+apt-get install -y \
+	apt-transport-https \
+	ca-certificates \
+	curl \
+	software-properties-common
+
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+apt-key fingerprint 0EBFCD88
```

Comment:
Can you define this as a variable at the top with a comment explaining what this fingerprint is?
Also, this script doesn't appear to be `set -e`, so this can fail and the script will continue fwics...?